src/lcm/LifeCycleStates.cc

```diff
@@ -71,7 +71,10 @@ void  LifeCycleManager::save_success_action(int vid)
 
         vm->get_requirements(cpu, mem, disk, pci);
 
-        hpool->del_capacity(vm->get_previous_hid(),vm->get_oid(),cpu,mem,disk,pci);
```

Comment:
This del_capacity comes from add_capacity in LifeCycleActions::244. This add action is always performed  (no check current==previous), so capacity will be added twice.

Tests need to include this check also (host capacity is preserved)